src/Microsoft.DotNet.Wpf/src/Shared/MS/Internal/Generated/ToleranceTypeValidation.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+//
+//
+// This file was generated, please do not edit it directly.
+//
+// Please see MilCodeGen.html for more information.
+//
+
+
+namespace System.Windows.Media
+{
+    internal static partial class ValidateEnums
+    {
+        /// <summary>
+        ///     Returns whether or not an enumeration instance a valid value.
+        ///     This method is designed to be used with ValidateValueCallback, and thus
+        ///     matches it's prototype.
+        /// </summary>
+        /// <param name="valueObject">
+        ///     Enumeration value to validate.
+        /// </param>    
+        /// <returns> 'true' if the enumeration contains a valid value, 'false' otherwise. </returns>
+        public static bool IsToleranceTypeValid(object valueObject)
+        {
+            ToleranceType value = (ToleranceType) valueObject;
+
+            return (value == ToleranceType.Absolute) || 
+                   (value == ToleranceType.Relative);
+        }                                
```

Comment:
@ThomasGoulet73 any ideas how come this file was not already present ?
This is dead code as this was not being used anywhere in the repo. But I guess we should leave it here for now.

Reply:
I'm not sure why the file wasn't present and it was never present in this repo. Some files in this folder are generated but never used so I think it's fine to generate this file and not use it.